src/main/java/com/oracle/truffle/bf/test/BFContext.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package com.oracle.truffle.bf.test;
+
+import com.oracle.truffle.api.ExecutionContext;
```

Comment:
You could try to do some interop with JavaScript by using Interop.eval(mimeType, script).